lab-michaelstuart/README.md

```diff
@@ -0,0 +1,13 @@
+# Lab 09 - persistent storage
```

Comment:
Excellent. Taking the time to work on the docs is really important. My docs game went to another level once I found dillinger.io which is a fantastic online markdown editor with great examples and a great preview window.